cicerone/parse/__init__.py

```diff
@@ -15,3 +9,10 @@
     "parse_spec_from_url",
     "parse_spec_from_yaml",
 ]
+
+# Re-export for backward compatibility
```

Comment:
I don't think this is necessary - despite what I said let's expose the functions in parser at this level so that users of the library can do this:

```
from cicerone import parse

parse.parse_spect_from_dict
```

etc